src/bidiMapper/domains/script/Realm.ts

```diff
@@ -171,7 +154,10 @@ export class Realm {
 
     if (deepSerializedValue.type === 'node') {
```

Comment:
Node type of `node` should throw if we try to evaluate them in a non `WindowRealm`, There is no DOM for other realms so no node should exist.

Reply:
In that case, it automatically throws in the CDP call.

Reply:
So this check should live on the WindowRealm and not on the Realm base class